src/app/components/server-charts/server-charts.component.ts

```
@@ -0,0 +1,791 @@
+      ?.forEach((data: any) =>
+      {
+      const name: string = data.name.replace('Geekbench:', '');
+      const desc = data.description.replace('The score is calibrated against a baseline score of 2,500 (Dell Precision 3460 with a Core i7-12700 processor) as per the Geekbench 6 Benchmark Internals.', '') || '';
+      this.geekbenchHTML += `<li> - ${name}: ${desc} </li>`;
+    });
+
+    this.geekbenchHTML += `</ul>`;
+
+    this.geekbenchHTML = this.sanitizer.bypassSecurityTrustHtml(this.geekbenchHTML);
+
+    if(dataSet && dataSet.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)